tests/v2/test_wakunode.nim

```diff
@@ -210,6 +210,54 @@ procSuite "WakuNode":
 
     await node1.stop()
     await node2.stop()
+  
+  asyncTest "Filter protocol works between light node and full node":
```

Comment:
I'd be careful with using words such as "light" and "full", I know we often take this shortcut but being more precise here would be useful. I.e. we are really talking about adaptive nodes, that have a varying set of capabilities.

Maybe we can specify what we mean by light and full here?

Also see https://github.com/vacp2p/rfc/issues/87 for some previousy discussion

Reply:
Ah, thanks, that discussion is very enlightening. Conceptually much cleaner to think in terms of capabilities.